### ISSUE TITLE
Build error when using rxjs-compat: All declarations of 'observable' must have identical modifiers.

### ISSUE THREAD

**Issue by vincent-cm** [ReactiveX/rxjs#3769]
## Bug Report

**Current Behavior**
A clear and concise description of the behavior.
ERROR in node_modules/rxjs/internal/symbol/observable.d.ts(4,9): error TS2687: All declarations of 'observable' must have identical modifiers.
node_modules/@types/node/index.d.ts(167,14): error TS2687: All declarations of 'observable' must have identical modifiers.

**Reproduction**
- REPL or Repo link:
(you can use https://stackblitz.com/ to create one to attach here)

```js
var your => (code) => here;
```

**Expected behavior**
A clear and concise description of what you expected to happen (or code).

**Environment**
- Runtime: [e.g. Node v${x}, Chrome v${x}]
- RxJS version:
- (If bug is related) Loader, build configuration: [e.g webpack, angular-cli version, config]

**Possible Solution**

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by cartant** [comment]
Dupe of #3697?

**Comment by kwonoj** [comment]
Looks likely by reading error message.

**Comment by cartant** [comment]
`@types/node` uses [the `readonly` modifier](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/index.d.ts#L166-L170):

```ts
interface SymbolConstructor {
    readonly observable: symbol;
    readonly iterator: symbol;
    readonly asyncIterator: symbol;
}
```

as does [`symbol-observable`](https://github.com/benlesh/symbol-observable/blob/master/index.d.ts#L5-L7):

```ts
declare global {
  export interface SymbolConstructor {
    readonly observable: symbol;
  }
}
```

The declaration in [`master`](https://github.com/ReactiveX/rxjs/blob/master/src/internal/symbol/observable.ts) does not. but once #3773 is merged, all of the declarations should be compatible.

**Comment by ulfryk** [comment]
#3773 merged, is there release ETA ?

**Comment by andreimcristof** [comment]
got this error. how to workaround until release please?

**Comment by andreimcristof** [comment]
@vincent-cm thanks, it worked. cheers

**Comment by cristifilip** [comment]
this has been fixed in 6.2.1 https://www.npmjs.com/package/rxjs/v/6.2.1

**Comment by cristifilip** [comment]
this has been fixed in 6.2.1 https://www.npmjs.com/package/rxjs/v/6.2.1

**Comment by adappter** [comment]
The problem is still present with latest versions:

```
    "rxjs": "6.2.1",
    "@types/node": "10.3.6"
```

```
ERROR in ../node_modules/rxjs/internal/symbol/observable.d.ts(4,9): error TS2687: All declarations of 'observable' must have identical modifiers.

node_modules/@types/node/index.d.ts(169,14): error TS2687: All declarations of 'observable' must have identical modifiers.
```

**Comment by cartant** [comment]
@adappter The declaration is `readonly observable: symbol;` in `@types/node` version 10.5.1 and is the same in `rxjs` 6.2.1.

You might want to update to the latest `@types/node` declarations and might also want to ensure that you have only one installation of `rxjs` in your `node_modules` - to do so, run `npm list rxjs`.

**Comment by adappter** [comment]
this is the output of npm list rxjs:
```
@angular-devkit/build-angular@0.6.8
│ ├─┬ @angular-devkit/architect@0.6.8
│ │ └── rxjs@6.2.1  deduped
│ └── rxjs@6.2.1  deduped
├─┬ @angular-devkit/core@0.6.8
│ └── rxjs@6.2.1  deduped
├─┬ @angular/cli@6.0.8
│ ├─┬ @angular-devkit/schematics@0.6.8
│ │ └── rxjs@6.2.1  deduped
│ ├─┬ @schematics/update@0.6.8
│ │ └── rxjs@6.2.1  deduped
│ └── rxjs@6.2.1  deduped
└── rxjs@6.2.1
```
I installed the latest @types/nodes 10.5.1. I can also see the same declarations in both files. But I still get the error. Also after reinstalled node modules

**Comment by cartant** [comment]
@adappter You should be able to verify, yourself, that the `@types/node` and `rxjs` declarations for `observable` symbol match. If they do, then your issue differs from this one. Perhaps there is another declaration of `observable` symbol somewhere in your code base.

In any case, if the two declarations are the same, you should open another issue instead of commenting on this issue - which is closed. As a general rule, it's better to create a new issue rather than comment on one that's closed. Just reference the closed issue from the newly opened one, if you suspect they are related.

**Comment by adappter** [comment]
Alright I'll have a look at will open a new issue if necessary
